autoload/SpaceVim/layers/edit.vim

```diff
@@ -211,16 +211,16 @@ function! s:text_transient_state() abort
         \ 'key' : 'J',
         \ 'desc' : 'move text down',
         \ 'func' : '',
-        \ 'cmd' : 'noautocmd normal! "_ddp',
+        \ 'cmd' : 'noautocmd silent! m .+1',
         \ 'exit' : 0,
         \ },
         \ ],
         \ 'right' : [
         \ {
         \ 'key' : 'K',
         \ 'func' : '',
-        \ 'desc' : 'move text up',
-        \ 'cmd' : 'noautocmd exe line(".") == 1 ? "" : "normal! \"_ddkP"',
+        \ 'desc' : 'noautocmd move text up',
```

Comment:
A type? I think the noautocmd should be added to `cmd` instead of `desc`.

Reply:
oh man, what did I do :joy: